docs/source/integrations/mlflow.md

```diff
@@ -0,0 +1,308 @@
+and {external+mlflow:ref}`the MLflow tracking server documentation <logging_to_a_tracking_server>`.
+:::
+
+## Simple use cases
+
+Although MLflow works best when working with machine learning (ML) and AI pipelines,
+you can track your regular Kedro runs as experiments in MLflow even if they do not use ML.
+
+This section explains how you can use the [`kedro-mlflow`](https://kedro-mlflow.readthedocs.io/) plugin
+to track your Kedro pipelines in MLflow in a straightforward way.
+
+### Easy tracking of Kedro runs in MLflow using `kedro-mlflow`
+
+To start using `kedro-mlflow`, install it first:
+
+```
+pip install kedro-mlflow
+```
+
+In recent versions of Kedro, this will already register the `kedro-mlflow` Hooks for you.
+
+Next, create a `mlflow.yml` configuration file in your `conf/local` directory
+that configures where the MLflow runs are stored,
+consistent with how you launched the `mlflow ui` command:
+
+```yaml
+server:
+  mlflow_tracking_uri: mlflow_runs
+```
+
+From this point, when you execute `kedro run` you will see the logs coming from `kedro-mlflow`:
+
+```
+[06/04/24 09:52:53] INFO     Kedro project spaceflights-mlflow                                     session.py:324
+                    INFO     Registering new custom resolver: 'km.random_name'                  mlflow_hook.py:65
+                    INFO     The 'tracking_uri' key in mlflow.yml is relative          kedro_mlflow_config.py:260
+                             ('server.mlflow_(tracking|registry)_uri = mlflow_runs').
+                             It is converted to a valid uri:
+                             'file:///Users/juan_cano/Projects/QuantumBlackLabs/kedro-
+                             mlflow-playground/spaceflights-mlflow/mlflow_runs'
+```
+
+If you open your tracking server UI you will observe a result like this:
+
+```{image} ../meta/images/complete-mlflow-tracking-kedro-mlflow.png
+:alt: Complete MLflow tracking with kedro-mlflow
+:width: 80%
+:align: center
+```
+
+Notice that `kedro-mlflow` used a subset of the `run_params` as tags for the MLflow run,
+and logged the Kedro parameters as MLflow parameters.
+
+Check out {external+kedro-mlflow:doc}`the official kedro-mlflow tutorial <source/03_getting_started/02_first_steps>`
+for more detailed steps.
+
+### Artifact tracking in MLflow using `kedro-mlflow`
+
+`kedro-mlflow` provides some out-of-the-box artifact tracking capabilities
+that connect your Kedro project with your MLflow deployment, such as `MlflowArtifactDataset`,
+which can be used to wrap any of your existing Kedro datasets.
+
+Use of this dataset has the advantage that the preview capabilities of the MLflow UI can be used.
+
+:::{warning}
+This will work for datasets that are outputs of a node,
+and will have no effect for datasets that are free inputs (hence are only loaded).
+:::
+
+For example, if you modify the a `matplotlib.MatplotlibWriter` dataset like this:
+
+```diff
+ dummy_confusion_matrix:
```

Comment:
While experienced users might find it obvious that you need to modify `catalog.yml` here, it may not be as clear to new users. Therefore, I suggest adding a comment here and next examples, similar to the one you included below, such as:
`# conf/base/catalog.yml`

Reply:
Good catch!